external-api/src/v1/controllers/estore/eStore.controller.ts

```diff
@@ -187,46 +217,36 @@ export const create = async (req: EstoreRequest, res: Response) => {
          * has been initiated.
          */
         if (siteCreationResponse?.data?.siteId) {
-          await eStoreSiteCreationCron(eStoreData);
+          await eStoreSiteCreationCronJob(eStoreData);
         } else {
-          console.error('eStore site creation failed for deal %s %o', eStoreData.dealIdentifier, siteCreationResponse?.data);
+          console.error('eStore site creation failed for deal %s %o', dealIdentifier, siteCreationResponse?.data);
 
           // CRON job log update
-          await cronJobLogs.updateOne(
-            { _id: { $eq: new ObjectId(_id) } },
-            {
-              $set: {
-                'cron.site.create': {
-                  response: siteCreationResponse?.data,
-                  status: ESTORE_CRON_STATUS.FAILED,
-                  timestamp: getNowAsEpoch(),
-                },
-                'cron.site.status': ESTORE_CRON_STATUS.FAILED,
-              },
+          await EstoreRepo.updateByDealId(eStoreData?.dealId, {
```

Comment:
Think this one has been missed, can be done in the next PR

Reply:
Ah! apologies have updated.